t/t0001-powerman-client.t

```diff
@@ -0,0 +1,22 @@
+#!/bin/sh
+
+test_description='Test Powerman Client without server'
+
+. `dirname $0`/sharness.sh
+
+powerman=$SHARNESS_BUILD_DIRECTORY/src/powerman/powerman
+
+ret="$?"
```

Comment:
this var isn't used?  i assume you removed some checks after discovering the issues you recently posted.